### ISSUE TITLE
Highlighting is broken if there is a comment after function parentheses

### ISSUE THREAD

**Issue by hudochenkov** [babel/babel-sublime#302]
Code to reproduce:

```js
/* @flow */
"use strict"
module.exports = function (
  statement/*: Object*/,
  options/*:: ?: Object*/
)/*: string*/ {
  options = options || {}

  let result = ""
  let rule/*: postcss$rule*/
  let atRule/*: postcss$atRule*/

  if (statement.type === "rule") {
    rule = statement
  }
  if (statement.type === "atrule") {
    atRule = statement
  }
  if (!rule && !atRule) {
    return result
  }
}
```

More examples at https://github.com/stylelint/stylelint/tree/master/lib/utils

The problem appears if there is no space between closing parenthesis and a comment:

![screen shot 2017-01-20 at 19 06 01](https://cloud.githubusercontent.com/assets/654597/22156409/2afd35fa-df44-11e6-9541-502bd9b24999.png)
![screen shot 2017-01-20 at 19 05 57](https://cloud.githubusercontent.com/assets/654597/22156421/3421c7b8-df44-11e6-85a6-d3a9d53e5a94.png)

If there is a space, everything is good:

![screen shot 2017-01-20 at 19 06 15](https://cloud.githubusercontent.com/assets/654597/22156502/948a12ae-df44-11e6-8798-dd84cf515c6b.png)

**Comment by borela** [comment]
Try FJSX15 https://github.com/borela/naomi

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
